conda_package_handling/tarball.py

```diff
@@ -100,7 +101,12 @@ def create(prefix, file_list, out_fn, out_folder=os.getcwd(), **kw):
                                                  out_fn.replace('.tar.bz2', ''),
                                                  '.tar.bz2', 'bzip2')
             final_path = os.path.join(out_folder, os.path.basename(out_file))
-            shutil.move(out_file, final_path)
+            try:
+                shutil.move(out_file, final_path)
+            except OSError as e:
```

Comment:
I wonder if [`atomicwrites`](https://python-atomicwrites.readthedocs.io/en/latest/#atomicwrites.move_atomic) wouldn't be more robust? I guess it couldn't do much about permission errors though...